src/Services/Air/Air.js

```diff
@@ -43,14 +43,14 @@ module.exports = (settings) => {
     },
 
     ticket(options) {
-      return service.importPNR(options).then((data) => {
+      return this.importPNR(options).then((data) => {
```

Comment:
Be careful because using `this` can cause troubles in test. Don't forget to mock service functions after that.